### PR TITLE
OpenNebula vmcontext.sh for FreeBSD guests.

### PR DESCRIPTION
**Pull request by olevole**
Tested on FreeBSD 10.0+

### CODE REVIEW

**Comment by FlorianHeigl** [comment]
also found this: https://github.com/slob/freebsd-one-context

**Comment by FlorianHeigl** [comment]
update: works better than what I use now, will try to make the best of both :)

File: share/scripts/freebsd/net-vmcontext/vmcontext [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+gen_exports() {
+	NETWORK=$1
+	echo "/images -network ${NETWORK}.0/24"
+}
+
+gen_hostname() {
+	MAC=$1
+	NUM=`mac2ip $MAC | cut -d'.' -f4`
+	NUM2=`echo 000000$NUM | sed 's/.*\(..\)/\1/'`
+	sysrc -q hostname="cluster${NUM2}" > /dev/null
```
Comment (FlorianHeigl) [inline share/scripts/freebsd/net-vmcontext/vmcontext]: This doesn't work; unfortunately you can't just assume the system is called "cluster_something"

**Comment by FlorianHeigl** [comment]
it is not fun to autoclose contribs instead of upstreaming them :-( :-(

**Comment by tinova** [comment]
Isn't this covered in the current context addon FreeBSD support?

https://github.com/OpenNebula/addon-context-linux

Let us know so we can close this.

**Comment by FlorianHeigl** [comment]
@olevole can you check please?

**Comment by tinova** [comment]
Please let us know if we should add something to the current FreeBSD support by OpenNebula.

**Pull request closed without merging**